examples/accesscontrol/podRoleBinding.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: app-service-account
+  namespace: tnf
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  namespace: tnf
+  name: pod-reader
+rules:
+  - apiGroups: [""]  # "" indicates the core API group
+    resources: ["pods"]
+    verbs: ["get", "watch", "list"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: read-pods
+  namespace: default
+subjects:
+  - namespace: tnf
+    kind: ServiceAccount
+    name: app-service-account
+roleRef:
+  # "roleRef" specifies the binding to a Role / ClusterRole
+  kind: Role  # this must be Role or ClusterRole
+  name: pod-reader  # this must match the name of the Role
+  apiGroup: rbac.authorization.k8s.io
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  creationTimestamp: null
+  labels:
+    run: rolebindingpod
+    app: test
+    test-network-function.com/generic: target
+    test-network-function.com/container: target
+  name: rolebindingpod
+  namespace: tnf
+spec:
+  # This should be set to false explicitly to satisfy testing requirements.
```

Comment:
Is this =  "automountServiceAccountToken" ?

Reply:
Good call, being explicit will make it clearer.  I'll adjust.